esm/FederPy/setup.py

```diff
@@ -4,16 +4,18 @@
     long_description = fh.read()
 
 setuptools.setup(
-    name="feder-py",
-    version="0.1.0",
+    name="federpy",
+    version="0.2.0",
```

Comment:
Do we need to make the node and python version consistent?

Reply:
The federpy imports the latest federjs automatically.
The version of federpy is only relevant to the python-interface.